File: exs/mundo_1/javascript/012.js
/*
Desafio 012

Problema: Faça um algoritmo que leia o preço de um produto e 
          mostre seu novo preço, com 5% de desconto

Resolução do problema:
*/

const input = require('readline-sync').question

let preco = Number(input('Informe o preço do produto: R$'))

total = preco*0.95

console.log(`Total à pagar com 5% de desconto: R$${total.toFixed(2)}`)

File: exs/mundo_1/rust/012.rs
/*
Desafio 012

Problema: Faça um algoritmo que leia o preço de um produto e
          mostre seu novo preço, com 5% de desconto.
*/

fn main() {
    let mut valor = String::new();

    println!("Informe o preço do produto (R$):");
    std::io::stdin().read_line(&mut valor).unwrap();

    let valor: f32 = valor.trim().parse().expect("Erro ao ler valor");

    let valor_desconto = valor * 0.95;

    println!("O valor do produto com desconto é: R${valor_desconto:.2}");
}
